tests/pom.xml

```diff
@@ -29,7 +29,7 @@
 					<useUIHarness>true</useUIHarness>
 					<useUIThread>false</useUIThread>
 					<application>org.eclipse.ui.ide.workbench</application>
-					<argLine>-Xms2048m -Xmx4096m</argLine>  
+					<argLine>-Dorg.eclipse.swt.internal.gtk.cairoGraphics=false -Xms2048m -Xmx4096m</argLine>  
 					<skipTests>${skipTests}</skipTests>
```

Comment:
I think this is already on the master

Reply:
yes they were actually cherry picked from here to another branch and then to master